src/main/java/uk/gov/hmcts/reform/sscs/callback/handlers/DwpUploadResponseHandler.java

```diff
@@ -55,11 +55,38 @@ public void handle(CallbackType callbackType, Callback<SscsCaseData> callback) {
             handleUc(callback);
         } else if (StringUtils.equalsIgnoreCase(benefitType.getCode(), Benefit.CHILD_SUPPORT.getShortName())) {
             handleChildSupport(callback);
+        } else if (StringUtils.equalsIgnoreCase(benefitType.getCode(), Benefit.TAX_CREDIT.getShortName())
+            || StringUtils.equalsIgnoreCase(benefitType.getCode(), Benefit.GUARDIANS_ALLOWANCE.getShortName())
+            || StringUtils.equalsIgnoreCase(benefitType.getCode(), Benefit.TAX_FREE_CHILDCARE.getShortName())
+            || StringUtils.equalsIgnoreCase(benefitType.getCode(), Benefit.HOME_RESPONSIBILITIES_PROTECTION.getShortName())
+            || StringUtils.equalsIgnoreCase(benefitType.getCode(), Benefit.CHILD_BENEFIT.getShortName())
+            || StringUtils.equalsIgnoreCase(benefitType.getCode(), Benefit.THIRTY_HOURS_FREE_CHILDCARE.getShortName())
+            || StringUtils.equalsIgnoreCase(benefitType.getCode(), Benefit.GUARANTEED_MINIMUM_PENSION.getShortName())
+            || StringUtils.equalsIgnoreCase(benefitType.getCode(), Benefit.NATIONAL_INSURANCE_CREDITS.getShortName())) {
+            handleSscs5Case(callback);
         } else {
             handleNonUc(callback);
         }
     }
 
+    private void handleSscs5Case(Callback<SscsCaseData> callback) {
```

Comment:
This looks very similar to the handleChildSupport method with lots of duplicated code. Can we not do something abit more tidier and move some of the code to reusable methods so we are not just copy and pasting?